src/itemstackmetadata.cpp

```diff
@@ -1,34 +1,57 @@
 #include "itemstackmetadata.h"
 #include "util/serialize.h"
 #include "util/strfnd.h"
+#include <json/json.h>
 
-#define DESERIALIZE_START '\x01'
-#define DESERIALIZE_KV_DELIM '\x02'
-#define DESERIALIZE_PAIR_DELIM '\x03'
-#define DESERIALIZE_START_STR "\x01"
-#define DESERIALIZE_KV_DELIM_STR "\x02"
-#define DESERIALIZE_PAIR_DELIM_STR "\x03"
+#define SERIALIZATION_VERSION_IDENTIFIER '\x02'
```

Comment:
I'm not sure if it's possible to write this to a (plain text) player file, needs testing

Reply:
Player data is saved to a database now (with my settings).
If you still want to use a common character, use `§`. Previous versions would encapsulate it. I suggest adding a version number to both `ItemStack` and `ItemStackMetadata` (so the serialized string would start with `§<version_number>`)